prime-router/docs/getting-started/README.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started as a ReportStream Developer
+
+## Overview
+
+### Tools to install
```

Comment:
This still needs to be addressed.

Reply:
There are, but the point of this ticket is to put the headers/files in so that tickets like that do not have to go searching for where to write content.